taipy/gui/gui.py

```diff
@@ -652,6 +660,10 @@ def _manage_message(self, msg_type: _WsType, message: dict) -> None:
             else:
                 _warn(f"Decoding Message has failed: {message}", e)
 
+    # To be expanded by inheriting classes
```

Comment:
Can you add a comment somewhere what the distinction 'internal' vs 'external' messages is?